src/lib/editor-routes.js

```diff
@@ -0,0 +1,20 @@
+import { Route, Redirect } from 'react-router';
+import React from 'react';
+import AppController from 'components/editor/AppController';
+import ArticleListController from 'components/editor/ArticleListController';
+import ArticleController from 'components/editor/ArticleController';
+import AuthorListController from 'components/editor/AuthorListController';
+import AuthorController from 'components/editor/AuthorController';
+
+export default (
+  <Route path="/" component={AppController}>
+    <Route path="articles/page/:page/" component={ArticleListController}>
+      <Route path="article/slug/:slug/" component={ArticleController} />
+      <Redirect to="article/slug/:slug" from="article/slug/:slug/" />
```

Comment:
It was just annoying that I couldn't delete part of the url and go to it when there wasn't a '/' at the end of the line, so made it redirect so it's not that strict. And no it's not an infinite loop.